cli/smartling/status.go

```diff
@@ -16,32 +16,38 @@ func MustStatus(remotefile, locale string) smartling.FileStatus {
 	return fs
 }
 
-type ProjectStatus map[string]map[string]smartling.FileStatus
+type ProjectStatus struct {
+	sync.RWMutex
+	internal map[string]map[string]smartling.FileStatus
+}
 
 func (statuses ProjectStatus) Add(remotefile, locale string, fs smartling.FileStatus) {
-	_, ok := statuses[remotefile]
+	statuses.Lock()
+	_, ok := statuses.internal[remotefile]
 	if !ok {
 		mm := make(map[string]smartling.FileStatus)
-		statuses[remotefile] = mm
+		statuses.internal[remotefile] = mm
 	}
-	statuses[remotefile][locale] = fs
+	statuses.internal[remotefile][locale] = fs
+	statuses.Unlock()
```

Comment:
Change these to a `defer statuses.Unlock()` call right after the lock call so any extra panics/returns added in this method don't break things